package.json

```diff
@@ -5,6 +5,7 @@
     "test": "echo \"There are no tests\" && exit 0"
   },
   "dependencies": {
-    "govuk-frontend": "^1.0.0"
+    "govuk-frontend": "^1.0.0",
+    "selenium-webdriver": "^4.0.0-alpha.1"
```

Comment:
try to delete this pls?

Reply:
I can't remember what we meant by this - delete the file or just those two dependancies?

Reply:
It was to remove the `"selenium-webdriver": "^4.0.0-alpha.1"`, as we didn't know what why it was introduced.

Reply:
yep, did it and it sorted out yarn.lock